core/block.js

```diff
@@ -1357,3 +1357,26 @@ Blockly.Block.prototype.moveBy = function(dx, dy) {
 Blockly.Block.prototype.makeConnection_ = function(type) {
   return new Blockly.Connection(this, type);
 };
+
+/**
+ * Checks whether all inputs are filled.
```

Comment:
/**
 * Recursively checks whether all statement and value inputs are filled with blocks.
 * Also checks all following statement blocks in this stack.
 * Shadow blocks count as filling an input.
 * @return ...
 */